README.md

```diff
@@ -110,6 +110,107 @@ python -m torch.distributed.launch --nproc_per_node 4 --master_port 9527 train.p
 python -m torch.distributed.launch --nproc_per_node 8 --master_port 9527 train_aux.py --workers 8 --device 0,1,2,3,4,5,6,7 --sync-bn --batch-size 128 --data data/coco.yaml --img 1280 1280 --cfg cfg/training/yolov7-w6.yaml --weights '' --name yolov7-w6 --hyp data/hyp.scratch.p6.yaml
 ```
 
+<a name="code_additions"/>
+
+#### Forked code additions
+This sections illustrates the changes added to main code in this fork.
+
+There are three main additions:
+1. Custom augmentations: This permits the addition of new augmentations not present in the ultralytics pipeline.
```

Comment:
yolov7 pipeline instead of ultralytics. Yolov7 is not published by ultralytics.

Reply:
I used the README I wrote for ultralytics as a base, I'll search for other instances